include/oc_api.h

```diff
@@ -1134,6 +1134,12 @@ void oc_resource_make_public(oc_resource_t *resource);
  */
 void oc_resource_set_discoverable(oc_resource_t *resource, bool state);
 
+
+#if defined(OC_PUSH) && defined(OC_SERVER) && defined(OC_CLIENT) && defined(OC_DYNAMIC_ALLOCATION) && defined(OC_COLLECTIONS_IF_CREATE)
+void oc_resource_set_pushable(oc_resource_t *resource, bool state);
```

Comment:
please add doxygen documentaton on the function

see also other header files

Reply:
done